eng/ci/templates/jobs/run-emulated-tests-windows.yml

```diff
@@ -1,9 +1,12 @@
+parameters:
+  poolName: ''
+
 jobs:
   - job: "TestWindows"
     displayName: 'Test java worker - Windows'
 
     pool:
-      name: 1es-pool-azfunc-public
+      name: ${{ parameters.poolName }}
```

Comment:
this had issue with being parameterised, I remember I tried this out for some other pipeline, we can skip setting it in the job may be and it should get picked from the stage?

Reply:
looks like it worked for the public pipeline, nvm.